docker/build_docker.sh

```diff
@@ -249,6 +249,43 @@ function docker_build {
             return 1
         fi
     fi
+
+    echo "================= $(date): Starting version verifocation on image: ${image_name} ================="
+    echo "Checking that python version is match to the base version..."
+    # Get the python version from the docker metadata.
+    PYTHON_VERSION=$(docker inspect "$image_full_name" | jq -r '.[].Config.Env[]|select(match("^PYTHON_VERSION"))|.[index("=")+1:]')
+    PY3CMD="python3"
+    if command -v python3.7 >/dev/null 2>&1; then
+        PY3CMD="python3.7"
+    elif command -v python3.8 >/dev/null 2>&1; then
+        PY3CMD="python3.8"
+    elif command -v python3.9 >/dev/null 2>&1; then
+        PY3CMD="python3.9"
+    elif command -v python3.10 >/dev/null 2>&1; then
+        PY3CMD="python3.10"
+    fi
+    if [ -f "Pipfile" ]; then
+        file_name="Pipfile"
+        pattern='python_version = \"([^\"]+)\"'
+        file_content=$(<"Pipfile")
+        if [[ $file_content =~ $pattern ]]; then
+            version_from_file="${BASH_REMATCH[1]}"
+        fi
+    fi
+    if [ -f "pyproject.toml" ]; then 
+        file_name="pyproject.toml"
+        pattern='python = \"([^\"]+)\"'
+        file_content=$(<"pyproject.toml")
+        if [[ $file_content =~ $pattern ]]; then
+            version_from_file="${BASH_REMATCH[1]}"
+        fi
+    fi
```

Comment:
Those 2 blocks of code are almost identical, maybe we should create a function, that based on the time of the file (Pipfile/poetry) returns the content and the pattern and then we could write the if statement just once.